uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/Service/AbsenceType.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+use CRM_HRLeaveAndAbsences_BAO_AbsenceType as AbsenceType;
+use CRM_HRLeaveAndAbsences_BAO_AbsencePeriod as AbsencePeriod;
+use CRM_HRLeaveAndAbsences_BAO_TOILRequest as TOILRequest;
+
+class CRM_HRLeaveAndAbsences_Service_AbsenceType {
+
+  /**
+   * Actions that occurs when an AbsenceType is updated
+   *
+   * @param \CRM_HRLeaveAndAbsences_BAO_AbsenceType $absenceType
+   */
+  public static function postUpdateActions(AbsenceType $absenceType) {
```

Comment:
Can you please make the methods on this service non static? The reasons is that, in the future, if we need to inject this service somewhere, we would be able to mock the methods